proposals/extensions_v2.md

```diff
@@ -0,0 +1,349 @@
+# Modern Extensions
+
+* [x] Proposed
+* [ ] Prototype: Not Started
+* [ ] Implementation: Not Started
+* [ ] Specification: Not Started
+
+Many thanks to those who helped with this proposal.  Esp. @jnm2!
+
+## Summary
+[summary]: #summary
+
+Modern Extensions introduce a new syntax to produce "extension members", greatly expanding on the set of supported members including properties, static methods and constructors, in a clean and cohesive fashion.
```

Comment:
Will this include operators? While I think we should push to initially preview this without operators, the final version of extensions likly needs to include them.

Reply:
Yes.  THis includes operators.